windows/sound.cpp

```diff
@@ -143,9 +144,9 @@ QString CSound::CheckDeviceCapabilities()
     if ( ( CanSaRateReturn == ASE_NoClock ) || ( CanSaRateReturn == ASE_NotPresent ) )
     {
         // return error string
-        return tr ( "The audio device does not support the "
-                    "required sample rate. The required sample rate is: " ) +
-               QString().setNum ( SYSTEM_SAMPLE_RATE_HZ ) + " Hz";
+        return QString ( tr ( "The selected audio device is incompatible "
+        "since it does not support a sample rate of %1 Hz. Please select another "
+        "device." ) ) .arg ( SYSTEM_SAMPLE_RATE_HZ );

```

Comment:
Do we need to convert SYSTEM_SAMPLE_RATE_HZ to a QString?

Reply:
No, QString is rendering the args for you.

Reply:
Great. Thanks. There are some places where it is converted.